src/Microsoft.IdentityModel.Tokens/TokenValidationResult.cs

```diff
@@ -35,6 +36,11 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public class TokenValidationResult
     {
+        /// <summary>
+        /// The <see cref="Dictionary{TKey, TValue}"/> created from the validated security token.
+        /// </summary>
+        public IDictionary<string, object> Claims { get; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Correct me if I am wrong, but claim names should be case sensitive, yes?

Either way, would be good to add a line to the summary or a remarks section that indicates this uses OrdinalIgnoreCase or Ordinal